vmware-base/src/com/cloud/hypervisor/vmware/mo/GuestOsDescriptorType.java

```diff
@@ -0,0 +1,46 @@
+// Licensed to the Apache Software Foundation (ASF) under one
+// or more contributor license agreements.  See the NOTICE file
+// distributed with this work for additional information
+// regarding copyright ownership.  The ASF licenses this file
+// to you under the Apache License, Version 2.0 (the
+// "License"); you may not use this file except in compliance
+// with the License.  You may obtain a copy of the License at
+//
+//   http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing,
+// software distributed under the License is distributed on an
+// "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+// KIND, either express or implied.  See the License for the
+// specific language governing permissions and limitations
+// under the License.
+package com.cloud.hypervisor.vmware.mo;
+
+public enum GuestOsDescriptorType
+{
+  windows9Guest,
+  windows9_64Guest,
+  windows9Server64Guest,
+  rhel7Guest,
+  rhel7_64Guest,
+  debian7Guest,
+  debian7_64Guest,
+  debian8Guest,
+  debian8_64Guest,
+  coreos_64Guest,
+  darwin12_64Guest,
+  darwin13_64Guest,
+  darwin14_64Guest;
+
+  public static GuestOsDescriptorType fromValue(String value) {
+    if (value == null) {
+      return null;
+    }
+    GuestOsDescriptorType guestOsType = null;
+    try {
+      guestOsType = valueOf(value);
+    } catch (IllegalArgumentException e) {
+    }
+    return guestOsType;
+  }
```

Comment:
Thanks @GabrielBrascher I checked that with latest VMware SDK I could remove this wrapper. I've pushed another PR on master that bumps up the vmware sdk version to 6.0. I'll remove this soon, and resend for a review.